certified-connectors/AXtension Content Gate/apiDefinition.swagger.json

```diff
@@ -326,6 +326,11 @@
                             "description": "The file type of the content entity.",
                             "title": "File Type"
                           },
+                          "fileTypeDisplayName": {
+                            "type": "string",
+                            "description": "The friendly name of the     file type of the content entity.",
```

Comment:
@JoepKillaars Why there is a long space between `The friendly name of the` and `file type of the content entity." ?